Makefile

```diff
@@ -39,6 +39,10 @@ test-reconciler:
 	sh openshift/e2e-rekt-tests.sh
 .PHONY: test-reconciler
 
+test-reconciler-keda:
+	sh test/keda-reconciler-tests.sh
+.PHONY: test-reconciler-keda
```

Comment:
Usually, upstream scripts don't work in midstream, we create custom ones https://github.com/openshift-knative/eventing-kafka-broker/blob/main/openshift/e2e-rekt-tests.sh